.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+name: Erlang CI
+
+on: [push]
+
+jobs:
+  build_and_test:
+    runs-on: ubuntu-latest
+    name: OTP ${{matrix.otp}}
+    strategy:
+      matrix:
+        otp: [23.2]
+    steps:
+    - uses: actions/checkout@v2.0.0
+    - uses: gleam-lang/setup-erlang@v1.1.2
+      with:
+        otp-version: ${{matrix.otp}}
+    - name: Compile
+      run: make compile
+    - name: Run xref
+      run: make xref
+    - name: Run dialyzer
+      run: make dialyze
+    - name: Run eunit tests
+      run: make eunit
+    - name: Run ct tests
+      run: make ct
```

Comment:
```suggestion
```

We don't have any CT tests in this library but I guess it does not hurt to still run the command..

Reply:
that's what I thought